symphony-bdk-core-invokers/symphony-bdk-core-invoker-jersey2/src/main/java/com/symphony/bdk/core/api/invoker/jersey2/ApiClientJersey2.java

```diff
@@ -205,6 +211,23 @@ public ApiClient setBasePath(String basePath) {
     return this;
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public ApiClient setClientCertificate(String certificatePassword, String password) throws GeneralSecurityException, IOException {
```

Comment:
can we keep the client final and have different constructors instead of mutating the object?